### ISSUE TITLE
Customize User-Facing Okta Touchpoints | Password Reset Flow

### ISSUE THREAD

**Issue by LGFranklin** [CDCgov/prime-simplereport#1327]
### Why are we doing this?
- Okta's out of the box solutions don't reflect many user experience best practices. Some of these result in a large volume of support requests. Addressing these pain points before we scale is critical to our efforts to well serve the most customers possible.
The pain points we intend to address are:
    - SMS requires an additional button press before a code is sent
    - On account creation, we currently ask users for phone number twice (once for PW reset, once for MFA)
    - The process had more security steps than needed
    - Once you create your account you land on an Okta screen and not in the app
    - A lot of support requests around password reset
- We believe that by easing users' ability to create, log in and recover accounts in a more self-serve manner, we will increase satisfaction and ongoing use.
- We are trying to balance a good user experience alongside security best practices and CDC security requirements. 

[Figma File](https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/CDC-USDS?node-id=1669%3A3)

### Summary
Implement designs for new password reset flow.

### Open questions
 - Design follows current path; if email is enabled, there will be more questions.

**Comment by timothybest-usds** [comment]
duplicate of https://github.com/CDCgov/prime-simplereport/issues/1331

**Comment by vz3** [comment]
The PW strength meter was pulled from Login.gov so we should draw from there for a lot of the interaction decisions.

They currently have following states:
- Red: "Weak" (Fail)
- Orange "So-so" (Fail)
- Light green "Good" (Pass)
- Dark green "Great!" (Pass)

For now can we have this be (cc @Rebecca-LM):
- Red: "Weak" (Fail, 0-1 criteria)
- Orange: "Weak" (Fail, 3 criteria)
- Yellow: "Okay" (Fail, 4 req)
- Green: "Good" (Pass, 5 reqs)

Struggling on the verbiage here (could maybe use some help from @Rebecca-LM). We may need to move to 3 bars (Red (Weak)/Yellow (Okay)/Green (Good).

https://github.com/18F/identity-idp
https://images.zenhubusercontent.com/302387839/ff134ab8-86ed-4d5d-ae1d-810eff82fd96/login_gov_meter.mov

**Comment by Rebecca-LM** [comment]
@vz3 Agree to move to 3 bars to keep it simple. My suggestion: "Weak" could be all criteria that fail, then "Medium" (encourages them to go beyond to get to the next level) and Pass is "Strong" (so it aligns with weak)

**Comment by maya** [comment]
@Rebecca-LM Everything fails (1-3 reqts) until all (4) are met. At what level would you use "Medium"?

**Comment by Rebecca-LM** [comment]
If we're using the 3 bars then medium would appear at two bars, getting them towards strong as they add letters/numbers or whatever they need. Does this make sense?

**Comment by maya** [comment]
No it doesn't make sense b/c we're using 4 bars.

**Comment by Rebecca-LM** [comment]
We discussed changing to only 3 bars b/c it doesn't add any valuable/new info to have 4

**Comment by maya** [comment]
Mentioned in chromatic, but we have 4 requirements: a bar for each one (4).

**Comment by Rebecca-LM** [comment]
In that case what do you think of Weak, Okay, Medium, Strong?

**Comment by emmastephenson** [comment]
Not implementing custom password reset at this time.
